src/components/Tokens/TokenTable/NetworkFilter.tsx

```diff
@@ -143,7 +144,7 @@ export default function NetworkFilter() {
       {open && (
         <MenuTimeFlyout>
           {BACKEND_CHAIN_NAMES.map((network) => {
-            const chainInfo = getChainInfo(CHAIN_NAME_TO_CHAIN_ID[network])
+            const chainInfo = getChainInfo(supportedChainIdFromGQLChain(network))
```

Comment:
so ideally `BACKEND_CHAIN_NAMES` is typed as `InterfaceGqlChain[]` and if you do the typescript function overloading thing mentioned in another comment for `supportedChainIdFromGQLChain`, then you `chainInfo` will always be defined and you won't have to do the null check on 148 and 172-173